src/garage/torch/algos/recurrent_encoder.py

```diff
@@ -0,0 +1,79 @@
+# pylint: disable=attribute-defined-outside-init
+"""A recurrent network with LSTM for encoding context of RL tasks."""
+
+import torch
+from torch import nn as nn
+
+from garage.torch.modules import MLPModule
+
+
+class RecurrentEncoder(MLPModule):
+    """This recurrent network encodes context of RL tasks.
+
+    Context is stored in the terms of observation, action, and reward, and this
+    network uses an MLP module followed by an LSTM model for encoding.
+
+    Args:
```

Comment:
hmm. this can cause maintainability problems. can you list the args/kwargs explicitly?